ios/development-rules.md

```diff
@@ -0,0 +1,426 @@
+# Development rules & recommendations
+
+- Here, you can find information about managing JIRA issues, git repos, GitHub pull requests, etc.
+- Mainly, you will find information about our codebase here
+
+## JIRA tasks
+- Our projects usually have a JIRA project associated with it
+- Creating issues and managing them can differ from project to project
+
+## Git & GitHub
+- We usually create a git branch for every JIRA issue, e.g. `git checkout -b feature/MUV-XXX-some-title` (`MUV` being the project code on JIRA)
+- Depending on whether the task corresponds to a fix or a feature, the branch usually has a prefix `fix/` or `feature/`
+- The branch name usually contains the JIRA issue number `MUV-XXX` so that CI/CD can automatically manage the issue state etc.
+- In order to keep your branch up-to-date with the base branch, we prefer `git rebase`
+- Try to follow the recommendations for git commit messages, such as writing them in do-forms and keeping them informative, very specific information [here](https://cbea.ms/git-commit/)
+- Once you are finished with the implementation, you can create a Pull Request on GitHub
+
+### GitHub Pull Requests
+- Try to stick with the naming style `[MUV-XXX] <Pull request title>`
+- We have a template for our pull request descriptions, which you will be able to fill out as soon as you create the pull request
+- Try to fill out the entire template accordingly:
+    - **Description**: Try to sum up what the PR introduces
+    - **What's new**: Describe what this PR changes
+    - **What's missing**: Try to sum up stuff related to the PR that you feel should be further developed or what you didn't have time for
+    - **References**: Add links to the stuff that your PR is related to, usually it's at least the JIRA issue and chili tasks, if available
+    - You can add more sections if you see fit.
+- Once you're done with the PR description, you can add reviewers – these would usually be the people who are working on the project with you
+- Always try to react to the reviewers' messages so that they can later see how you handled them
+- When you're done with all the review comments, request a re-review, so that the reviewers can see that you've fixed all their comments and that the code is ready for a re-review
+- Once the PR is ready to be merged, it should be merged by you, the PR creator
+
+## Architecture
+
```

Comment:
Remove newline to keep the consistent style pls

Reply:
✅